### PR TITLE
#164688423 A user can update an answer

### PR DESCRIPTION
**Pull request by IssaIan**
### What does this PR do?
Adds edit answer functionality
### What are the tasks to be completed?
```
- Ensure that a user can update an existing answer that they posted
```
### Local Development Setup
 - First Create python virtual env
 ```
  $ virtualenv -p python .venv
 ```
 - Install Requirements
 ```
  $ pip install -r requiremts.txt
 ```
 - Copy .env-example to .env and set config
 ```
  $ cp .env-example .env
 ```
 - Create postgres database
 ```
  $ sudo su postgres
  $ psql
  postgres=# CREATE USER django WITH PASSWORD 'password';
  postgres=# ALTER ROLE django SET client_encoding TO 'utf8';
  postgres=# ALTER ROLE django SET default_transaction_isolation TO 'read committed';
  postgres=# ALTER ROLE django SET timezone TO 'UTC';
  postgres=# CREATE DATABASE drf;
  postgres=# GRANT ALL PRIVILEGES ON DATABASE drf TO django;
  postgres=# \q
  $ exit
````
### Create admin user
Create admin user using your email and password
```
1. cd api/
2. python manage.py createsuperuser
 ```
 - Run Server
 ```
  $ python manage.py runserver
 ```
a. **After cloning the repo:-**
```
$ git checkout  ft-edit_answer-164688423

```
b. **Run the server. On postman create a meetup, create a question, then create an answer. Then send the following sample data with a PUT method on the url `http://127.0.0.1:8000/api/meetups/<meetupId>/questions/<questionId>/answers/<answerId>`**
```
{
	"body": "test update answer"
}

```
### Relevant Pivotal Tracker Story
[#164688423](https://www.pivotaltracker.com/n/projects/2315729/stories/164688423)

### Screenshots
n

### CODE REVIEW

**Review by Bakley — APPROVED** [review]
Good job @IssaIan . You have done an exemplary job

**Review by Ogutu-Brian — APPROVED** [review]
Good work @IssaIan

**Review by Amoswachira — APPROVED** [review]
looks good

File: api/config/swagger.py [hunk 1]
```diff
@@ -415,6 +444,37 @@ def schema_view_swagger(request):
                     ],
                     description='Fetching questions on a specific meetup'
                 ),
+                'update existing answer': coreapi.Link(
+                    url='/api/meetups/{meetupId}/questions/{questionId}/answers/{answerId}',
+                    action='PUT',
+                    fields=[
+                        coreapi.Field(
+                            name='questionId',
```
Comment (Amoswachira) [inline api/config/swagger.py]: can you change the name form questionid to meetupid? since you have to two fields with the same name

**Review by Amoswachira — CHANGES_REQUESTED** [review]

File: api/config/swagger.py (hunk 1 above)
Comment (IssaIan) [reply api/config/swagger.py]: Made the requested changes

**Review by Amoswachira — APPROVED** [review]

**Review by Ogutu-Brian — CHANGES_REQUESTED** [review]
Hello @IssaIan ,
Fix the merge conflicts

**Comment by IssaIan** [comment]
Hello @davenjeru. This branch closed as the changes were merged into develop from another pull request which is the one for consolidating google login and signup url endpoints.

**Pull request closed without merging**